doc/build-osx.md

```diff
@@ -20,23 +20,23 @@ Dependencies
 
 See [dependencies.md](dependencies.md) for a complete overview.
 
-If you want to build the disk image with `make deploy` (.dmg / optional), you need RSVG
+If you want to build the disk image with `make deploy` (.dmg / optional), you need RSVG.
 
     brew install librsvg
 
 Berkeley DB
 -----------
 It is recommended to use Berkeley DB 4.8. If you have to build it yourself,
 you can use [the installation script included in contrib/](/contrib/install_db4.sh)
-like so
+like so:
 
 ```shell
 ./contrib/install_db4.sh .
 ```
 
 from the root of the repository.
 
-**Note**: You only need Berkeley DB if the wallet is enabled (see the section *Disable-Wallet mode* below).
+**Note**: You only need Berkeley DB if the wallet is enabled (see the section *Disable-wallet mode* below).
```

Comment:
**Note**: You only need Berkeley DB if the wallet is enabled (see the section [*Disable-wallet mode*](/doc/build-osx.md#disable-wallet-mode) below).

Reply:
Make section name as a link.